docs/Modding/index.rst

```diff
@@ -56,7 +56,7 @@ Have a look at :file:`amplio.tilespec` and associated files in :file:`/amplio` f
 guides document specific aspects of tileset creation:
```

Comment:
I was thinking that the index could be skipped (fewer clicks to reach the target page!) since it essentially repeats the information appearing in this section. Proper structure is guaranteed by the `toctree` appearing under the "Tilesets" heading.

If only one page is envisioned here, better link it in a more visible way and revise the sentence introducing it to be in singular form.

Reply:
Need to do the same for Rulesets then too. Another PR for that tho.